packages/gestalt/src/Masonry.js

```diff
@@ -41,6 +47,9 @@ type Props<T> = {|
   items: Array<T>,
   measurementStore: Cache<T, *>,
   minCols: number,
+  // Content layer and Viewport layer is as defined in Collection.
+  onLayerUpdate?: (contentLayer: Layer, viewportLayer: Layer) => void,
+  onPendingMeasurementsUpdate?: (hasPendingMeasurements: boolean) => void,
```

Comment:
This is almost a double negative. It would be made clearer by not using a boolean for the arg and instead thinking of the state being an enum. `onAutoMeasuringUpdate?: (state: 'idle' | 'measuring')`.